Describe the PieChart - annotations demo (#252)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/Annotation/description.md

> Annotations are images and text blocks that provide additional information on the visualized data.

Reviewer: Just another version... Again, the reason is the word "provide". 

```suggestion
Annotations are images and text blocks that add information to the visualized data.
```

> To configure individual annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/) option. Each object configures an individual annotation. To specify options for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over these settings.

Reviewer: I made a change to a similar block in another merge request. I don't know why you add the word "individual" if you then switch to an "array". Then the next sentence talks about "individual" annotations again (repetition). 

Here's another option: 

```suggestion
To create annotations, populate the widget's [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/) array. Each object in the array configures an individual annotation. To specify options for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
```

> Each annotation should have the [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#type) set to *"text"* or *"image"*. Depending on the **type**, specify the [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#text) or [image](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/image/) option. You can also add a [description](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#description) that will be displayed in a tooltip.

Reviewer: Please see my other comments about this part (the word "should").

Annotations are containers for images, text blocks, and custom content that display additional information about the visualized data.

To create annotations, populate the widget's [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/) array. Each object in the array configures an individual annotation. To specify options for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.

> You can set each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#type) attribute to *"text"*, *"image"*, or *"custom"*. Depending on the **type**, specify the [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#text) or [image](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/image/) option. You can also add a [description](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#description) that will be displayed in a tooltip.

Reviewer 2: ```suggestion
You can set each annotation's [type](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#type) option to *"text"*, *"image"*, or *"custom"*. Depending on the **type**, specify the [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#text) or [image](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/image/) option. You can also add a [description](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/annotations/#description) that will be displayed in a tooltip.
```
